docs/guides/mdx-on-demand.server.mdx

```diff
@@ -28,7 +28,11 @@ On the server:
 ```js path="server.js"
 import {compile} from '@mdx-js/mdx'
 
-const code = String(await compile('# hi', {outputFormat: 'function-body' /* …otherOptions */ }))
+const code = String(await compile('# hi', {
+  outputFormat: 'function-body',
+  development: false, // Avoid use of the jsxDEV runtime
```

Comment:
I’m thinking maybe something like this? It’s a bit annoying, because it’s all very situational.

```suggestion
  // development: false, // Use this to avoid use of jsx-dev-runtime during development
```

Maybe for the client we can use:

```js
import {run} from '@mdx-js/mdx'
import * as runtime from 'react/jsx-runtime'
import * as devRuntime from 'react/jsx-dev-runtime'

const code = '' // To do: get `code` from server somehow.
const deveopment = process.env.NODE_ENV // To do: determine development mode for your setup.

const {default: Content} = await run(code, development ? devRuntime : runtime)
```